src/app/app.component.ts

```diff
@@ -2,20 +2,31 @@ import { DOCUMENT } from '@angular/common';
 import { Component, Inject, OnInit } from '@angular/core';
 import { IMenuItem, menuRoutes, MenuType } from './pages/menu.routes';
 import { PermissionController } from './services/permission.controller';
+import  feedbackfin from "feedbackfin";
+import {getLittilConfigFromWindow} from "../littilConfig";
+
+const littilConfig = getLittilConfigFromWindow();
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
 })
 export class AppComponent implements OnInit {
   public menuRoutes: IMenuItem[] = menuRoutes;
+  window:any;
 
   constructor(
     public readonly permissionController: PermissionController,
     @Inject(DOCUMENT) public document: Document
   ) {}
 
   public ngOnInit(): void {
+    this.window = this.document.defaultView
+    this.window.feedbackfin = { ...feedbackfin };
```

Comment:
I'd hope that this could work with `@ViewChild`. I'll see whether I can make that change before merging this, since that could be more reliable. Otherwise this is fine.

Reply:
Checked it, seem this plugin by default relies on accessing `window`, but doesn't need it. There is an error in the console however (both with the current version and with removing the use of `window`). So I'll approve for now, but maybe we can see whether that error can be resolved.

Reply:
Also added a commit that removes the use of the class property where a local variable is appropriate and that sets the config before loading the script.